meta-jaspar/recipes-jaspar/jaspar-image/jaspar-image.bb

```diff
@@ -1,6 +1,6 @@
 SUMMARY = "Main Jaspar image"
 
-IMAGE_FEATURES += "splash package-management x11-base x11-sato ssh-server-dropbear hwcodecs"
+IMAGE_FEATURES += "splash package-management x11-base x11-sato hwcodecs"
```

Comment:
Just so you know, this change will break ssh for the emulator and add it to the image. Well, at least that's my understanding. Shouldn't matter too much since it's currently not building for the emulator. This change is useful for development but we probably wouldn't want to leave it running by default on production.